contrib/src/main/java/com/datatorrent/contrib/cassandra/CassandraStore.java

```diff
@@ -115,21 +130,35 @@ public Session getSession() {
   /**
    * Creates a cluster object.
    */
-  public void buildCluster(){
-
+  public void buildCluster()
+  {
     try {
-
-      cluster = Cluster.builder()
-          .addContactPoint(node).withCredentials(userName, password).build();
-    }
-    catch (DriverException ex) {
+      if (protocolVersion != null && protocolVersion.length() != 0) {
+        ProtocolVersion version = getCassandraProtocolVersion();
+        cluster = Cluster.builder().addContactPoint(node).withCredentials(userName, password).withProtocolVersion(version).build();
+      } else {
```

Comment:
which version does it use in the default case? Should we add a specific default? Like for example, the latest version?